src/chrome/content/rules/am730.com.hk.xml

```diff
@@ -0,0 +1,28 @@
+<!--
+	Non-functional hosts
+		Couldn't resolve host name:
+			 - am730.com.hk
+			 - mailserv.am730.com.hk
+			 - www1.am730.com.hk
+
+		Timeout was reached:
+			 - epaper.am730.com.hk
+
+		Secure connection redirects to plaintext:
+			 - archive.am730.com.hk
```

Comment:
Please also add a comment for `flipbook.am730.com.hk` (serves different content)